keel-clouddriver/src/main/kotlin/com/netflix/spinnaker/keel/clouddriver/model/AmazonLoadBalancer.kt

```diff
@@ -24,6 +25,4 @@ interface AmazonLoadBalancer {
   val vpcId: String
   val subnets: Set<String>
   val scheme: String?
-  val idleTimeout: Int
-  val securityGroups: Set<String>
```

Comment:
Had to move these fields off the interface as NLBs don't have them. Didn't require any other changes.